queries/bundleAll.sql

```diff
@@ -6,23 +6,26 @@ SELECT clips.id,
   COALESCE(SUM(NOT votes.is_valid), 0) AS down_votes,
   COALESCE(age, '') AS age,
   COALESCE(gender, '') AS gender,
-  client_accent_list.accent_list as accents,
+  COALESCE(client_accent_list.accent_list, '') as accents,
   locales.name AS locale,
   COALESCE(terms.term_name, '') AS segment
 FROM clips
   LEFT JOIN votes ON clips.id = votes.clip_id
   LEFT JOIN user_client_accents accents ON clips.client_id = accents.client_id
   AND accents.locale_id = clips.locale_id
   -- A subquery that makes list of individual users accents
-  JOIN (
-    SELECT uc.client_id,
+  LEFT JOIN (
+    SELECT uca.client_id,
+      uca.locale_id,
       GROUP_CONCAT(a.accent_name) as accent_list
-    FROM user_clients uc
-      JOIN user_client_accents uca ON uc.client_id = uca.client_id
-      JOIN accents a ON uca.accent_id = a.id
-    WHERE a.accent_name != 'unspecified' and a.accent_name != ''
-    ORDER BY a.accent_name
-  ) client_accent_list ON accents.client_id = client_accent_list.client_id
+    FROM user_client_accents uca
+      JOIN accents a ON a.id = uca.accent_id
+    WHERE a.accent_name != 'unspecified'
+      and a.accent_name != ''
+    GROUP BY uca.locale_id,
+      uca.client_id
+  ) client_accent_list ON clips.client_id = client_accent_list.client_id
+  and client_accent_list.locale_id = clips.locale_id
```

Comment:
Join accent data by locale id with clips locale, that way only relevant accents are joined for the particular user